### PR TITLE
allow for title inside releases.json *Breaking Change

### PR DESCRIPTION
**Pull request by jbadan**
Prints provided title of each transform as it runs 
screen shot 2018-06-12 at 9 51 17 am

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import flatten from "lodash.flatten";
```
Comment (bcullman) [inline src/index.js]: https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore#_flatten

File: readme.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# pkg-upgrader 
+# pkg-upgrader
 
 <img src="screenshot.gif" width="440" align="right">
```
Comment (bcullman) [inline readme.md]: is it worth re-doing the animated gif here to show cli improvements?

Comment (benmonro) [reply readme.md]: 👍

**Comment by benmonro** [comment]
Also I think add a com at the end of your print so it's a bit more clear

**Comment by benmonro** [comment]
^colon. Not com

File: readme.md (hunk 2 above)
Comment (jbadan) [reply readme.md]: Do we want a gif of nui-widgets-codemods on public github or will that just be more confusing?

Comment (benmonro) [reply readme.md]: Definitely no.  Just a generic package

Comment (benmonro) [reply readme.md]: Definitely no.  Just a generic package

Comment (jbadan) [reply readme.md]: Adding an issue to do that :)

**Pull request merged**